guide/src/wasm-bindgen-test/browsers.md

```diff
@@ -0,0 +1,69 @@
+# Testing in Headless Browsers
+
+## Configure Your Test Crate
+
+Add this to the root of your test crate, e.g. `$MY_CRATE/tests/wasm.rs`:
+
+```rust
+wasm_bindgen_test_configure!(run_in_browser);
```

Comment:
This may want to use `use wasm_bindgen_test::wasm_bindgen_test_configure;` just above it (to bring the macro into scope)